pkg/rancher-desktop/main/mainmenu.ts

```diff
@@ -181,10 +181,10 @@ function getPreferencesMenuItem(): MenuItemConstructorOptions[] {
     {
       label:               'Preferences',
       visible:             true,
-      registerAccelerator: true,
+      registerAccelerator: false,
       accelerator:         'CmdOrCtrl+,',
       click() {
-        openMain(true);
+        openPreferences();
       },
```

Comment:
@rak-phillip I have a little concern about this lines. What is the benefit to call `openMain(true)` ? IMO there aren't, that's why I replaced it with `openPreferences()`.

Reply:
There's some history tied to this one.. All the preferences that we have today used to be in the main window, interspersed with application behavior. When we pulled all the preferences out into a separate window, we were left without a way to easily open the main window, so we opted to open both the main window along with the preferences window in as a short-term fix until we could come up with an appropriate pattern.

Reply:
We should keep this as `openMain(true)` until we can agree on an approach with #3722.